lib/theme.json

```diff
@@ -365,6 +389,34 @@
 						}
 					}
 				}
+			},
+			"core/post-comments-form": {
+				"elements": {
+					"textInput": {
+						"spacing": {
+							"padding": {
+								"top": "calc(0.667em + 2px)",
+								"right": "calc(0.667em + 2px)",
+								"bottom": "calc(0.667em + 2px)",
+								"left": "calc(0.667em + 2px)"
```

Comment:
I wonder if we should align these values between the blocks now that we don't have many and the form block is still experimental. The search block is probably the trickiest one since any style changes will break the inline version of the block /cc @jasmussen for visibility

Reply:
Thank you for the ping. Yes, those values are awkward, and best I recall it's to ensure the padding of the solid button matches height and footprint with the button that's outlined/has a stroke. 

Question: this PR, does that allow us to completely change those values, or are there still some back-compatibility concerns to consider? 

And CC: @WordPress/gutenberg-design for a broader set of input on what to potentially do here. Perhaps even @iamtakashi and @henriqueiamarino as block theme design experts might have input on best practices for styling elements like these, and can propose some great new defaults.